File: assignment 10 - C# practice problems/16.cs
//Write a c# function to check whether a string is blank or not.

using System;
namespace MyNameSpace
{
    class Program16
    {
        public static void CheckIfStringContainsBlankCharecter()
        {

            Console.WriteLine("Enter a string ");
            string stringInput1 = Console.ReadLine();


            if (stringInput1.Length == 0)
            {

                Console.WriteLine("String is blank");

            }
            else
            {
                Console.WriteLine("String is not blank");
            }


        }
    }
}

File: assignment 3 - js practice problems/web-dev-starter/16.js
//Write a JavaScript function to check whether a string is blank or not.
function checkIfStringContainsBlankCharecter() {
    let stringInput = prompt("Enter a string");
    if (stringInput != null) {
        stringInput = stringInput.trim();

        if (stringInput.length === 0) {
            alert("It is a blank String");
        } else {
            alert("It is not a blank String");
        }

    }
}

checkIfStringContainsBlankCharecter();